### PR TITLE
Add support  for processor API

### PR DESCRIPTION
**Pull request by shahbaz-mehar**

### CODE REVIEW

**Review by BushraAsif — COMMENTED** [review]
Changes reviewed

File: src/Response/Embeds/RedirectResponse.php [hunk 1]
```diff
@@ -27,8 +27,34 @@
...
+
+    /**
+     * @var Item
+     */
+    public $Data;
+
+    /**
+     * @var string
+     */
+    public $FlowType;
```
Comment (emicha) [inline src/Response/Embeds/RedirectResponse.php:59]: Aren't you over complicated the regular redirect response? `FlowType` is something you will never find anywhere else besides the processor API. Therefore you should be extending the response, not modifying it.

File: src/Response/Embeds/Item.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Copyright (c) 2016 Martin Aarhof
```
Comment (emicha) [inline src/Response/Embeds/Item.php:3]: Is this really the case?

File: src/Response/InitiatePaymentResponse.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright (c) 2016 Martin Aarhof
```
Comment (emicha) [inline src/Response/InitiatePaymentResponse.php:3]: Is this really the case?

File: src/Response/VerifyThreeDSecureResponse.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright (c) 2016 Martin Aarhof
```
Comment (emicha) [inline src/Response/VerifyThreeDSecureResponse.php:3]: Is this really the case?

File: src/Response/Embeds/Item.php (hunk 2 above)
Comment (shahbaz-mehar) [reply src/Response/Embeds/Item.php]: Updated the copyright with AltaPay and the current year.

File: src/Response/InitiatePaymentResponse.php (hunk 3 above)
Comment (shahbaz-mehar) [reply src/Response/InitiatePaymentResponse.php]: Updated the copyright with AltaPay and the current year.

File: src/Response/VerifyThreeDSecureResponse.php (hunk 4 above)
Comment (shahbaz-mehar) [reply src/Response/VerifyThreeDSecureResponse.php]: Updated the copyright with AltaPay and the current year.

**Pull request merged**